src/js/panels.js

```diff
@@ -1076,6 +1089,7 @@ https://github.com/fluid-project/first-discovery/raw/master/LICENSE.txt
             onError: null
         },
         listeners: {
+            "afterRender.savePrefs": "{that}.savePrefs",
```

Comment:
This save is to send POST request to the prefs server for saving prefs there and requesting a token. onSave is for saving prefs into the store. 

I probably should use a better name for this action. What about "savePrefsToServer"?